ReactiveXComponent/WebSocket/WebSocketClient.cs

```diff
@@ -111,10 +114,50 @@ private void WebSocketOnMessageReceived(object sender, MessageReceivedEventArgs
 
         public bool IsOpen { get { return _webSocket != null && _webSocket.State == WebSocketState.Open; } }
 
-        public void Init(WebSocketEndpoint endpoint, int timeout)
+        public void Init(WebSocketEndpoint endpoint, TimeSpan timeout, TimeSpan? retryInterval, int maxRetries)
         {
             _endpoint = endpoint;
             _timeout = timeout;
+
+            _maxRetries = maxRetries;
+            _retryInterval = (retryInterval != null) ? retryInterval.Value : TimeSpan.FromSeconds(5);

```

Comment:
It could be interesting to increment the _retryInterval depending on _currentRetryCount